Bio/Sequencing/Applications/__init__.py

```diff
@@ -29,6 +29,7 @@
     "BwaSampeCommandline",
     "BwaBwaswCommandline",
     "BwaMemCommandline",
+    "NovoalignCommandline",
```

Comment:
Good catch. Having the classes implemented tucked away like this has its downsides.